lib/pure/os.nim

```diff
@@ -1353,7 +1362,7 @@ when not weirdTarget:
         raise newException(ValueError, "The specified root is not absolute: " & root)
       joinPath(root, path)
 
-proc normalizePath*(path: var string) {.rtl, extern: "nos$1", tags: [], noNimScript.} =
+proc normalizePath*(path: var string) {.rtl, extern: "nos$1", tags: [].} =
```

Comment:
Are we certain this works in a NimScript environment?

Reply:
yes; I've just added a test to that effect